about.html

```diff
@@ -142,7 +174,78 @@ <h1 class="overlay-name">Spencer Perry</h1>
 				<img class="square" src="bubble-demo/1.jpg">
 			</div>
 		</div>
+	
+
+
+
+		<div class="our-services-page>
 		
+		<h1 class="services-header">Our Services</h1>
+
+<div class="services">
+	
+
+	<div class="app">
+			
+			<div class="popup-window" id="services-popup">
+				<a href="javascript:void(0)" class="close" onclick="closePopup()">&times;</a>
+				<div class="popup-window-content">
+					<img src="file:///Users/Dada/Desktop/mobile.svg">
+					<h2 style="line-height: 70%">App<br></br>Development</h2>
+					<p style="line-height: 100%"><span style="font-weight: bold">Mobile apps for both iOS and Android.</span><br></br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p>
+				</div>
+			</div>
+
+		<div class="popup" onclick="openPopup()">
+			<img src="file:///Users/Dada/Desktop/mobile.svg"></div>
```

Comment:
this is an absolute path to your file directory and breaks when anyone else tries to load the page. put the SVG icons into the img folder under assets and change this path to reflect that